### PR TITLE
add partner flag to api_key, add connection between api_key and organ…

### PR DESCRIPTION
**Pull request by axelsson**
…ization, add endpoint for external partners to access emissions/organizations data they should have access to

* New bool `partner` on api_keys to keep track of which api_keys belongs to our partners
* New joint table connecting api_keys to organizations
* New admin view for administering those connections
* New api endpoint for allowing external partners to access emissions data and which organizations they have access to

### CODE REVIEW

File: app/controllers/api/v1/partner/organizations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+module Api
+  module V1
+    module Partner
+      class OrganizationsController < ApiController
+        include EmissionsHelper
+        before_action :authorize_and_set_username
+
+        def show
+          key = params['key']
```
Comment (kallenilver) [inline app/controllers/api/v1/partner/organizations_controller.rb:11]: Lite förvirrande med key, apikey och username. Kan key döpas om till organization_key istället?

Comment (kallenilver) [reply app/controllers/api/v1/partner/organizations_controller.rb:11]: och @username kanske kan bli @api_key?

**Review by kallenilver — APPROVED** [review]
Bra jobbat! Allt verkar funka tycker jag!

File: app/controllers/api/v1/partner/organizations_controller.rb (hunk 1 above)
Comment (axelsson) [reply app/controllers/api/v1/partner/organizations_controller.rb]: Bra tänkt, har ändrat nu!

**Pull request merged**